packages/browser/README.md

```diff
@@ -37,12 +41,33 @@ const options: BacktraceConfiguration = {
 }
 
 // Initialize the client
-const client = BacktraceClient.builder(options).build();
+const client = BacktraceClient.initialize(options);
 
 // Send an error
 client.send(new Error("Something broke!"));
 ```
 
+## Sending an Error
+By default, Backtrace will send an error for Uncaught Exceptions and Unhandled Promise Rejections.
+
+### Manually Sending an Error
+There are several ways to send an error to Backtrace. For more details on the definition of ```client.send()``` see [Methods](#methods) below.
+
+```ts
+// send as a string
+await client.send('This is a string!');
```

Comment:
Maybe instead of this we can add here info
"this is the message report"?

Reply:
My only concern is it might be confusing calling is a report when we have examples creating actual instances of BacktraceReport below. Thoughts?